hknweb/candidate/migrations/0015_auto_20191119_2206.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 2.1.11 on 2019-11-20 06:06
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
```

Comment:
Uhh I don't remember making a migration not sure what this is

Reply:
I ran `makemigrations` on my computer and this re-appeared; it looks like you simply forgot to commit this migration in a much older change.